doc/source/ray-observability/state/state-api.rst

```diff
@@ -103,10 +133,22 @@ You can get the state of a single task using the get API.
 
 You can also access logs through ``ray logs`` API.
 
-.. code-block:: bash
+.. tabbed:: CLI
 
-    ray list actors
-    ray logs --actor-id <ACTOR_ID>
+    .. code-block:: bash
+
+        ray list actors
+        ray logs --actor-id <ACTOR_ID>
+
+.. tabbed:: Python SDK
+
+    .. code-block:: python
+
+        from ray.experimental.state.api import get_log
+
+        actor_id = "31405554844820381c2f0f8501000000"
```

Comment:
I think this <ACTOR_ID> vs concrete actor_id value thing is not consistent across all examples.

Can you just unify it to <ACTOR_ID> format? Like 

```
for line in get_log(address="http://localhost:8265", actor_id=<ACTOR_ID>): # In this case, ACTOR_ID is 31405554844820381c2f0f8501000000
```